kornia/contrib/face_detection.py

```diff
@@ -12,8 +11,33 @@
 
 __all__ = ["FaceDetector", "FaceDetectorResult", "FaceKeypoint"]
 
+NET_TYPES = ["yunet_n", "yunet_s"]
 
-url: str = "https://github.com/kornia/data/raw/main/yunet_final.pth"
+
+class BackboneConfig(TypedDict):
```

Comment:
Any opinion about dataclass vs typed dict In these configs? related to #2093

Reply:
I think that makes sense :+1: